lib/querytables.js

```diff
@@ -246,7 +236,7 @@ function getTablesMetadata (pg, tableArray, callback) {
         SELECT * FROM cdb_table_metadata ORDER BY reloid;`;
 
     const readOnly = true;
-    pg.query(metadataQuery, {}, (err, result) => {
+    pg.query(metadataQuery, {}, async (err, result) => {
```

Comment:
`pg` is not able to handle `async` functions as callbacks. I know it works because pg doesn't do anything with the promise that an `async` function returns. It may explode in the future when we change `pg`.

To resolve this situation: extract the method where you need to use `await` and the chain to it bot `then` and `catch` callbacks.

Something like this:

```js
pg.query(metadataQuery, {}, (err, result) => {
  [...]
  setUpdateAtToForeingTables(rows)
    .then(tablesMetadata => callback(null, tablesMetadata))
    .cathc(err => callback(err))
}, readOnly);

async function setUpdateAtToForeingTables (rows) {
  for (const row of rows) {
    if (row.relkind === 'f') {
      row.updated_at = await getForeignTableUpdatedAtPromise(pg, row);
    }
  }
  return rows;
}
```
Don't worry if something goes wrong `async` wraps any throw or rejection on `await` into a promise rejected for free.

Reply:
--- EDIT ---
To resolve this situation: extract the method where you need to use `await` and then chain it to both `then()` and `catch()` callbacks.